platforms/enterprise/enterprise/src/main/java/org/gradle/internal/enterprise/impl/DefaultGradleEnterprisePluginAdapter.java

```diff
@@ -55,7 +48,6 @@ public class DefaultGradleEnterprisePluginAdapter implements GradleEnterprisePlu
     private final DefaultGradleEnterprisePluginServiceRef pluginServiceRef;
 
     private final BuildOperationNotificationListenerRegistrar buildOperationNotificationListenerRegistrar;
-    private final Multimap<Throwable, Problem> problems;
```

Comment:
💭 @mlopatkin For the future, I suppose the right approach would be to pass the necessary data about problems to `GradleEnterprisePluginAdapter.buildFinished()` instead of trying to store it in the plugin adapter itself, right?

Reply:
> For the future, I suppose the right approach would be to pass the necessary data about problems to `GradleEnterprisePluginAdapter.buildFinished()` instead of trying to store it in the plugin adapter itself, right?

Either this, or have a service here that can do the lookup rather than storing the mapping data itself.